tests/test_cli.py

```diff
@@ -52,6 +53,29 @@ def objective_func_multi_objective(trial: Trial) -> Tuple[float, float]:
     return (x + 5) ** 2, (x - 5) ** 2
 
 
+def _parse_output(output: str, output_format: str) -> Any:
+
+    if output_format == "table":
+        rows = output.split("\n")
+        assert all(len(rows[0]) == len(row) for row in rows)
+        assert rows[0] == rows[2] == rows[-1]
```

Comment:
In my understanding, this line checks the ruled lines of the table. How about adding a code comment?
```suggestion
        # Check ruled lines.
        assert rows[0] == rows[2] == rows[-1]
```